config/admit-wh.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
I'm not a big fan of keeping this dynamically generated configuration here. There should be one source of truth and that's `kudo init...` for now. Maybe we can rework `init` command to produce individual manifests and then reuse them?

Reply:
we already have CRDS under `config` @zen-dog how is this different?

Reply:
CRDS are **not** dynamically generated and the `/config/crds` **is** the source of truth for them. This is different for the manifests.

Reply:
@zen-dog `/config/crds` is NOT the source of truth...  they are generated from the source with `make generate`

Reply:
you're right, it's not 🙈 thx, for clarifying. but then `/config/crds` is just a remnant from the times where we needed to generate the CRDs for the integration tests. these days, we could simply run `kudo init ...` as the first step and avoid having them there. is there a reason why you need these new manifests added to the git repo?